packages/b/brscan4/files/brscan4.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="driver">
+  <id>brscan4</id>
+  <name>Brscan4 drivers</name>
+  <summary>Brscan4 drivers for Brother scanners</summary>
+  <description>
+    <p>
+      Brscan4 provides a variety of scanning drivers for various Brother scanners
+      and multi-function printers.
+    </p>
+    <p>
+      Note that some portions are binary and provided for hardware enablement
+      so may not meet typical Solus packaging standards for paths.
+    </p>
+  </description>
+  <url type="homepage">https://global.brother</url>
+  <icon type="stock">brother</icon>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary:BROTHER</project_license>
+  <developer_name>Brother Industries</developer_name>
+  <requires>
+    <kernel version="9.99" compare="eq">Linux</kernel>
+  </requires>
```

Comment:
Is this required for all driver?

Reply:
Nah you can drop that. That only gets edited if the driver depends upon a specific kernel e.g. nvidia